### PR TITLE
Adds jest-styled-components to EditorialFeature

### PR DESCRIPTION
**Pull request by kierangillen**
- Adds `jest-styled-components` to EditorialFeature test and updates snapshot

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 63e20d6318c18819ddd688f044b2900d74935388

https://deploy-preview-2045--artsy-reaction.netlify.com

**Comment by mzikherman** [comment]
Dope!

**Comment by artsyit** [comment]
:rocket: PR was released in v14.0.7 :rocket:

File: src/Components/Publishing/EditorialFeature/__tests__/EditorialFeature.test.tsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import { mount } from "enzyme"
+import "jest-styled-components"
```
Comment (damassi) [inline src/Components/Publishing/EditorialFeature/__tests__/EditorialFeature.test.tsx]: Should this be added to [jest_setup](https://github.com/artsy/reaction/blob/master/src/setup_jest.ts)? // cc @ds300

**Pull request merged**